### DISCUSSION TITLE
Trying to add instance using a shortcut to the game folder

### DISCUSSION THREAD

**Discussion by PoofImAlex** [KSP-CKAN/CKAN#4167] [Q&A] [answered]
i cannot import ksp into the software, i have the right version installed but its just taking too long and sometimes saying catastrophic error
any way someone can help?

**Comment by PoofImAlex** [comment]
![image](https://github.com/user-attachments/assets/e85cbbc7-581f-4ae6-b95d-7830f5188074)
ive been waiting for a while aswell

**Comment by HebaruSan** [comment]
Hi @PoofImAlex, what exactly is taking too long? It's not clear what you mean by "import ksp into the software". What are you clicking?

**Reply by PoofImAlex** [reply]
i go to import it, i go to program files and click twice on ksp
then, the folder software closes but ksp doesnt appear

**Reply by PoofImAlex** [reply]
and it still says loading registry

**Reply by HebaruSan** [reply]
What are you clicking when you "go to import it" ?

What are you clicking that you think of as "on ksp" ?

**Reply by PoofImAlex** [reply]
i go to desktop file and double click on ksp, it has the regular picture for the games logo
im clicking on new game instance
![image](https://github.com/user-attachments/assets/666b9412-d0d5-4a1a-87b0-ec1088c65952)
![image](https://github.com/user-attachments/assets/735525f4-1d1e-4b9d-8388-fdfe09d707f8)

**Reply by PoofImAlex** [reply]
even after that though, it doesnt appear
![image](https://github.com/user-attachments/assets/abfafb76-ae1f-45d5-be8a-a7a7eb4e13f0)

**Reply by PoofImAlex** [reply]
keep in mind this is my first time using this

**Reply by HebaruSan** [reply, marked as answer]
OK, clicking that shortcut will not work. You have to find the actual folder where your game is installed.

**Reply by PoofImAlex** [reply]
oh ok
thanks

**Reply by PoofImAlex** [reply]
its working fine thanks bro

**Reply by HebaruSan** [reply]
You're welcome, and thanks for reporting this. I did not know that this popup showed shortcuts; it shouldn't, since they don't match the filename filter that we give it. I'm going to see if there's anything we can do about that...

**Reply by PoofImAlex** [reply]
thanks to you i can play multiplayer :D

**Reply by HebaruSan** [reply]
@PoofImAlex that's great news, enjoy!

I can't force that popup to hide shortcuts, but I can show this message if you try to select one:

![image](https://github.com/user-attachments/assets/6338e37c-4ecc-4d81-a3ba-ec0133eeb740)

If that had appeared while you were trying to get started, would you have been able to figure it out?

**Reply by PoofImAlex** [reply]
yes, i probably would

**Reply by PoofImAlex** [reply]
nvm i cant play multiplayer, i have no friends
